main.go

```diff
@@ -106,7 +106,7 @@ func main() {
 		var status int
 		var functionResult []byte
 
-		statusCode := res.StatusCode
+		var statusCode int
 		if err != nil {
 			statusCode = http.StatusServiceUnavailable
 		}
```

Comment:
Shouldn't the else be on this line?

Reply:
actually 		
if err != nil {
	statusCode = http.StatusServiceUnavailable
}
these three line can be removed